build fail on docker

## Bug Report

Previous version is fine. 
After ' Update libuv to v1.42.0 and update NPM deps d55a65c' version happened. 

git clone https://github.com/versatica/mediasoup-demo.git
cd mediasoup-demo
git checkout v3
cd server
docker/build.sh

npm install in macOS is fine, but seems not work on linux. 
the error message paste on the last. 


### Your environment

- Operating system: macOS Big Sur 11.4
- Node version:v14.17.3
- npm version:6.14.13
- gcc/clang version:Apple clang version 12.0.5 
- mediasoup version:V3
- mediasoup-client version:


### Issue description
```

>>> Running docker build 
[+] Building 148.8s (9/13)                                                                                                                                                                                                   
 => [internal] load build definition from Dockerfile                                                                                                                                                                    0.0s
 => => transferring dockerfile: 37B                                                                                                                                                                                     0.0s
 => [internal] load .dockerignore                                                                                                                                                                                       0.0s
 => => transferring context: 2B                                                                                                                                                                                         0.0s
 => [internal] load metadata for docker.io/library/node:10                                                                                                                                                              0.0s
 => [internal] load build context                                                                                                                                                                                       0.0s
 => => transferring context: 4.18kB                                                                                                                                                                                     0.0s
 => CACHED [1/9] FROM docker.io/library/node:10                                                                                                                                                                         0.0s
 => [2/9] RUN  set -x  && apt-get update  && apt-get install -y net-tools build-essential valgrind                                                                                                                     27.6s
 => [3/9] WORKDIR /service                                                                                                                                                                                              0.0s 
 => [4/9] COPY package.json .                                                                                                                                                                                           0.0s 
 => ERROR [5/9] RUN npm install                                                                                                                                                                                       121.1s 
------                                                                                                                                                                                                                       
 > [5/9] RUN npm install:                                                                                                                                                                                                    
#9 3.916 npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.                                                                                           
#9 6.150 npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.                                                                                
#9 11.24 npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated                                                                                                                             
#9 11.25 npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated                                                                                                                                
#9 51.90 
#9 51.90 > bufferutil@4.0.3 install /service/node_modules/bufferutil
#9 51.90 > node-gyp-build
#9 51.90 
#9 52.05 
#9 52.05 > utf-8-validate@5.0.5 install /service/node_modules/utf-8-validate
#9 52.05 > node-gyp-build
#9 52.05 
#9 52.19 
#9 52.19 > heapdump@0.3.15 install /service/node_modules/heapdump
#9 52.19 > node-gyp rebuild
#9 52.19 
#9 53.61 make: Entering directory '/service/node_modules/heapdump/build'
#9 53.61   CXX(target) Release/obj.target/addon/src/heapdump.o
#9 54.28   SOLINK_MODULE(target) Release/obj.target/addon.node
#9 54.34   COPY Release/addon.node
#9 54.34 make: Leaving directory '/service/node_modules/heapdump/build'
#9 54.41 
#9 54.41 > mediasoup@3.7.19 postinstall /service/node_modules/mediasoup
#9 54.41 > node npm-scripts.js postinstall
#9 54.41 
#9 54.48 npm-scripts.js [INFO] running task "postinstall"
#9 54.48 npm-scripts.js [INFO] executing command: make -C worker
#9 54.48 make: Entering directory '/service/node_modules/mediasoup/worker'
#9 54.49 /usr/bin/python2 ./scripts/configure.py -R mediasoup-worker
#9 54.56 ['-R', 'mediasoup-worker', '/service/node_modules/mediasoup/worker/mediasoup-worker.gyp', '-I', '/service/node_modules/mediasoup/worker/common.gypi', '--depth=.', '-f', 'make', '-Goutput_dir=/service/node_modules/mediasoup/worker/out', '--generator-output', '/service/node_modules/mediasoup/worker/out', '-Dgcc_version=63', '-Dclang=0', '-Dhost_arch=x64', '-Dtarget_arch=x64', '-Dopenssl_fips=', '-Dmediasoup_asan=false', '-Dnode_byteorder=little']
#9 54.80 make -j6 BUILDTYPE=Release -C out
#9 54.81 make[1]: Entering directory '/service/node_modules/mediasoup/worker/out'
#9 54.81   CC(target) /service/node_modules/mediasoup/worker/out/Release/obj.target/libsrtp/deps/libsrtp/srtp/srtp/ekt.o
#9 54.81   CC(target) /service/node_modules/mediasoup/worker/out/Release/obj.target/libsrtp/deps/libsrtp/srtp/srtp/srtp.o
......
......
#9 118.1   CXX(target) /service/node_modules/mediasoup/worker/out/Release/obj.target/mediasoup-worker/src/main.o
#9 119.1   LINK(target) /service/node_modules/mediasoup/worker/out/Release/mediasoup-worker
#9 119.4 /service/node_modules/mediasoup/worker/out/Release/obj.target/deps/libuv/libuv.a(core.o): In function `uv_run':
#9 119.4 /service/node_modules/mediasoup/worker/out/../deps/libuv/libuv/src/unix/core.c:389: undefined reference to `uv__io_poll'
#9 119.4 /service/node_modules/mediasoup/worker/out/Release/obj.target/deps/libuv/libuv.a(core.o): In function `uv__io_close':
#9 119.4 /service/node_modules/mediasoup/worker/out/../deps/libuv/libuv/src/unix/core.c:956: undefined reference to `uv__platform_invalidate_fd'
#9 119.4 /service/node_modules/mediasoup/worker/out/Release/obj.target/deps/libuv/libuv.a(poll.o): In function `uv_poll_init':
#9 119.4 /service/node_modules/mediasoup/worker/out/../deps/libuv/libuv/src/unix/poll.c:74: undefined reference to `uv__io_check_fd'
#9 119.4 /service/node_modules/mediasoup/worker/out/Release/obj.target/deps/libuv/libuv.a(poll.o): In function `uv__poll_stop':
#9 119.4 /service/node_modules/mediasoup/worker/out/../deps/libuv/libuv/src/unix/poll.c:108: undefined reference to `uv__platform_invalidate_fd'
#9 119.4 /service/node_modules/mediasoup/worker/out/../deps/libuv/libuv/src/unix/poll.c:108: undefined reference to `uv__platform_invalidate_fd'
#9 119.4 /service/node_modules/mediasoup/worker/out/../deps/libuv/libuv/src/unix/poll.c:108: undefined reference to `uv__platform_invalidate_fd'
#9 119.4 /service/node_modules/mediasoup/worker/out/../deps/libuv/libuv/src/unix/poll.c:108: undefined reference to `uv__platform_invalidate_fd'
#9 119.4 /service/node_modules/mediasoup/worker/out/../deps/libuv/libuv/src/unix/poll.c:108: undefined reference to `uv__platform_invalidate_fd'
#9 119.4 /service/node_modules/mediasoup/worker/out/Release/obj.target/deps/libuv/libuv.a(linux-core.o): In function `uv__platform_loop_init':
#9 119.4 /service/node_modules/mediasoup/worker/out/../deps/libuv/libuv/src/unix/linux-core.c:90: undefined reference to `uv__epoll_init'
#9 119.4 /service/node_modules/mediasoup/worker/out/../deps/libuv/libuv/src/unix/linux-core.c:90: undefined reference to `uv__epoll_init'
#9 119.4 collect2: error: ld returned 1 exit status
#9 119.4 mediasoup-worker.target.mk:248: recipe for target '/service/node_modules/mediasoup/worker/out/Release/mediasoup-worker' failed
#9 119.4 make[1]: *** [/service/node_modules/mediasoup/worker/out/Release/mediasoup-worker] Error 1
#9 119.4 make[1]: Leaving directory '/service/node_modules/mediasoup/worker/out'
#9 119.4 make: *** [default] Error 2
#9 119.4 Makefile:24: recipe for target 'default' failed
#9 119.4 make: Leaving directory '/service/node_modules/mediasoup/worker'
#9 120.9 npm WARN notsup Unsupported engine for supports-color@9.0.2: wanted: {"node":">=12"} (current: {"node":"10.24.1","npm":"6.14.12"})
#9 120.9 npm WARN notsup Not compatible with your version of node/npm: supports-color@9.0.2
#9 120.9 npm WARN notsup Unsupported engine for has-flag@5.0.1: wanted: {"node":">=12"} (current: {"node":"10.24.1","npm":"6.14.12"})
#9 120.9 npm WARN notsup Not compatible with your version of node/npm: has-flag@5.0.1
#9 120.9 npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/chokidar/node_modules/fsevents):
#9 120.9 npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
#9 120.9 
#9 120.9 npm ERR! code ELIFECYCLE
#9 120.9 npm ERR! errno 1
#9 121.0 npm ERR! mediasoup@3.7.19 postinstall: `node npm-scripts.js postinstall`
#9 121.0 npm ERR! Exit status 1
#9 121.0 npm ERR! 
#9 121.0 npm ERR! Failed at the mediasoup@3.7.19 postinstall script.
#9 121.0 npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
#9 121.0 
#9 121.0 npm ERR! A complete log of this run can be found in:
#9 121.0 npm ERR!     /root/.npm/_logs/2021-07-21T02_39_46_910Z-debug.log
------
executor failed running [/bin/sh -c npm install]: exit code: 1

```

--- a/worker/deps/libuv/uv.gyp
+++ b/worker/deps/libuv/uv.gyp
@@ -245,6 +245,7 @@
             'libuv/src/unix/procfs-exepath.c',
             'libuv/src/unix/random-getrandom.c',
             'libuv/src/unix/random-sysctl-linux.c',
+            'libuv/src/unix/epoll.c',
           ],
           'link_settings': {
             'libraries': [ '-ldl', '-lrt' ],
@@ -261,6 +262,7 @@
             'libuv/src/unix/procfs-exepath.c',
             'libuv/src/unix/random-getrandom.c',
             'libuv/src/unix/random-sysctl-linux.c',
+            'libuv/src/unix/epoll.c',
           ],
           'link_settings': {
             'libraries': [ '-ldl' ],
@@ -358,6 +360,7 @@
             'libuv/src/unix/pthread-fixes.c',
             'libuv/src/unix/os390.c',
             'libuv/src/unix/os390-syscalls.c'
+            'libuv/src/unix/os390-proctitle.c'
           ]
         }],
       ]

